test/e2e/storage/persistent_volumes-local.go

```diff
@@ -266,6 +292,58 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 			framework.DeletePodOrFail(config.client, config.ns, pod2.Name)
 		})
 	})
+
+	Context("when using local volume provisioner", func() {
+		var (
+			volumePath string
+		)
+
+		BeforeEach(func() {
+			setupLocalVolumeProvisioner(config, node0)
+			volumePath = path.Join(
+				hostBase, discoveryDir, fmt.Sprintf("vol-%v", string(uuid.NewUUID())))
+		})
+
+		AfterEach(func() {
+			cleanupLocalVolumeProvisioner(config, node0, volumePath)
+		})
+
+		It("should create and recreate local persistent volume", func() {
+			By("Creating bootstrapper pod to start provisioner daemonset")
+			createBootstrapperPod(config)
+			kind := schema.GroupKind{Group: "extensions", Kind: "DaemonSet"}
+			framework.WaitForControlledPodsRunning(config.client, config.ns, daemonSetName, kind)
+
+			By("Creating a directory under discovery path")
+			framework.Logf("creating local volume under path %q", volumePath)
+			mkdirCmd := fmt.Sprintf("mkdir %v", volumePath)
+			runLocalUtil(config, node0.Name, mkdirCmd, hostBase)
+			oldPV, err := waitForLocalPersistentVolume(config.client, volumePath)
+			Expect(err).NotTo(HaveOccurred())
+
+			// Create a persistent volume claim for local volume: the above volume will be bound.
+			By("Creating a persistent volume claim")
+			claim, err := config.client.Core().PersistentVolumeClaims(config.ns).Create(newLocalClaim(config))
+			Expect(err).NotTo(HaveOccurred())
+			err = framework.WaitForPersistentVolumeClaimPhase(
+				v1.ClaimBound, config.client, claim.Namespace, claim.Name, framework.Poll, 1*time.Minute)
+			Expect(claim.Spec.VolumeName).To(Equal(oldPV.Name))
```

Comment:
Done.  I've appended namespace suffix to storage class name.

@jeffvance thanks for suggestion; but here, PVs are created from local volume provisioner, i can't control labels on PVs.